src/contexts/Drive/Drive.context.tsx

```diff
@@ -46,6 +48,52 @@ interface DriveContextProviderProps {
 const logger = new BaseLogger({
   tag: 'DRIVE_CONTEXT',
 });
+
+export const removeOldItemsFromLocalDB = async () => {
```

Comment:
maybe getModifiedDriveItems or something like that could explain better what this function does?

Reply:
The thing is that the function also remove the old cached item🤔

Reply:
I'll see what I can do

Reply:
`getModifiedDriveItemsAndUpdateLocalCache` could be a better name then, but up to you